### ISSUE TITLE
Software Boutique does not read button labels when using screenreader

### ISSUE THREAD

**Issue by saivinob** [ubuntu-mate/ubuntu-mate-welcome-legacy#90]
When using Software Boutique with screen reader (orca), it just says 'push button' for each tab press instead of reading the button's label.

How to reproduce:
1) Login to Ubuntu MATE (tested on 20.04.02 and Rolling Rhino)
2) Start screen reader with `Alt+Super+s`
3) Launch Software Boutique
4) Hit tab to navigate

Issue was [reported](https://ubuntu-mate.community/t/software-boutique-unlabled-buttons-with-orca-screenreader/23822) by a user on UM community forum.

Bug report link https://pad.lv/1921879
